build.gradle

```diff
@@ -87,8 +87,17 @@ void setupAllTargetsWithDefaultSourceSets(Project project) {
 
 private void setupIosTargets(Project project) {
     project.kotlin {
-        targets.fromPreset(presets.iosX64, 'iosX64')
-        targets.fromPreset(presets.iosArm64, 'iosArm64')
+        ios()
+        iosSimulatorArm64()
+
+        sourceSets {
+            iosSimulatorArm64Main {
+                dependsOn iosMain
+            }
+            iosSimulatorArm64Test {
+                dependsOn iosTest
+            }
+        }
```

Comment:
can you make sure you really need this?

Reply:
We do, that's exactly what our KMP DSL plugin does and what JetBrains recommend: https://kotlinlang.org/docs/mpp-share-on-platforms.html#use-target-shortcuts cc @minaEweida